buildmodel/commands.go

```diff
@@ -457,8 +461,63 @@ func EnsureDockerfileGenerationPrerequisites() error {
 // any auto-update code runs to detect problems before wasting time on an incompletable update.
 func RunDockerfileGeneration(repoRoot string) error {
 	fmt.Println("Generating Dockerfiles...")
-	var dockerfileUpdateScript = filepath.Join(repoRoot, "eng", "update-dockerfiles.sh")
-	return run(exec.Command("bash", dockerfileUpdateScript))
```

Comment:
For comparison, here's a link to this script that I'm essentially inlining then adding features to in this PR: https://github.com/microsoft/go-images/blob/microsoft/main/eng/update-dockerfiles.sh. (I'm on the fence about linking to the bash script from a code comment--I figure it makes sense not to, since I plan to delete the script in the move to submodules.)